### DISCUSSION TITLE
Documentation for IMS PRINT_OPTION  ALL outputs

### DISCUSSION THREAD

**Discussion by RyanConway91** [MODFLOW-ORG/modflow6#1265] [General]
I cannot find any documentation for what is getting written if PRINT_OPTION  ALL is specified

For example, what is BACKTRACK FLAG?

Can this get added to the mf6io.pdf?

**Comment by christianlangevin** [comment]
Have you read through this MODFLOW 6 Framework report?  It has information on the numerical solution.  https://pubs.er.usgs.gov/publication/tm6A57

**Comment by oscarfasanchez** [comment]
I have the same feeling, there is little information of the list file in the documentation. (for instance MODFLOW NWT has some information in Appendix A). For example, I don't know what means  *PTC in some iterations of the listing file(Pseudotransient continuation I think is active in every iteration, so I don't know what means its presence in some of them).
